aws-amplify/amplify-android
# feat(Auth): Add TOTP Support

*Issue #, if available:*

*Description of changes:*
* Add TOTP Support

*How did you test these changes?*
* manually tested various flows
* Added Unit Test

*Documentation update required?*
- [X] Yes (will updated later)

*General Checklist*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
